src/main/java/fr/cnes/sonarqube/plugins/framac/sensor/FramaCSensor.java

```diff
@@ -6,6 +6,10 @@
 import fr.cnes.sonarqube.plugins.framac.report.FramaCReportReader;
 import fr.cnes.sonarqube.plugins.framac.rules.FramaCRulesDefinition;
 import fr.cnes.sonarqube.plugins.framac.settings.FramaCPluginProperties;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.nio.charset.StandardCharsets'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonar-frama-c-plugin&pullRequest=23&issues=AWY0GieNM63is4RD-5M7&open=AWY0GieNM63is4RD-5M7)

[ISSUE_KEY]: AWY0GieNM63is4RD-5M7